component/site/views/default/helpers/defaultloadedfromtemplate.php

```diff
@@ -1027,14 +1027,14 @@ function DefaultLoadedFromTemplate($view, $template_name, $event, $mask, $templa
 						}
 						if (stripos($fieldval, "%d") !== false)
 						{
-							$days = intval($timedelta / (60 * 60 * 24));
+							$days = (int) $timedelta / (60 * 60 * 24);
```

Comment:
It maybe but it's better practice, technically although the improvement is small it's 6 times more efficient on average.